wurst/_handles/Item.wurst

```diff
@@ -15,6 +15,10 @@ public function item.remove()
 public function item.getTypeId() returns int
 	return GetItemTypeId(this)
 
+/** Returns "itemtype" of this item. Do not confuse it with itemTypeId. */
+public function item.getItemType() returns itemtype
```

Comment:
As `itemtype` is a type, can we call this `getItemtype()`? It will also aide in avoiding the concern you mention in the comment.